### DISCUSSION TITLE
Settings page with subpages

### DISCUSSION THREAD

**Discussion by zt64** [adrielcafe/voyager#237] [Q&A] [answered]
Hi. I recently switched to Voyager from my previous navigation setup and so far it's been fine but one thing I'm struggling with is how to migrate my settings. Previously I had a root settings screen, that would display all the categories. Tapping a category will then replace the content on that screen with the categories content. That way I would only have one Scaffold, topbar, etc. I can't immediately see how to do this with Voyager. Any ideas?

**Comment by Syer10** [comment, marked as answer]
It looks like you want neated navigators, just have a root screen and a Navigator inside the Scaffold. Where the default screen is a screen with all the categories. 
https://voyager.adriel.cafe/navigation/nested-navigation#nested-navigators
